verify.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, configfile, auth):
             elif opts[line] == '-d':
                 self.desc = opts[line + 1]
             elif opts[line] == '-b':
-                self.bin = opts[line + 1]
+                self.bin = self.desc
```

Comment:
Hi Nick the idea was to set both the config.bin and config.desc when the -d line is encountered in looping through the config.  What is the best practice for me to update the PR?  I'm not sure why the comments aren't coming through as I've been commenting directly in the Github web interface.